submissions/usides/friends-app/js/main.js

```diff
@@ -0,0 +1,159 @@
+const HUMANS_LINK =
+  'https://randomuser.me/api/?nat=us&results=36&inc=gender,name,picture,dob&seed=06b64d62bc01d624a';
+const BAD_STATUS = 200;
```

Comment:
Maybe it's a typo here because 200 is not a bad status code

Reply:
Oops. My bad. Fixed.